rebasehelper/application.py

```diff
@@ -439,6 +440,9 @@ def build_packages(self):
             results_dir = os.path.join(self.results_dir, version)
             build_dict['builds_nowait'] = self.conf.builds_nowait
             build_dict['build_tasks'] = self.conf.build_tasks
+            build_dict['enable_option'] = self.conf.enable_option
+            if self.conf.buildtool in ("copr", "fedpkg"):
+                logger.warning("We are not supporting option --enable-option for builder copr or fedpkg.")
```

Comment:
Well i think it's better to use warning cause it should inform you that this is not working on other builders and yes there is missing and with checking enable_option ... 

For me it's good. User forgets that enable_option cant be used for copr and fedpkg and he should be informed. If you want it through debug. I can do it through debug...